src/transformation.js

```diff
@@ -409,25 +398,23 @@ class TransformationBase {
    * @return PlainObject
    */
   toHtmlAttributes() {
-    var attrName, height, key, options, ref2, ref3, value, width;
-    options = {};
-    for (key in this.otherOptions) {
-      value = this.otherOptions[key];
-      if (contains(Transformation.PARAM_NAMES, snakeCase(key))) {
-        continue;
+    let options = {};
+    Object.keys(this.otherOptions).forEach((key) => {
```

Comment:
why not `for key in this.otherOptions` instead? Object.keys().forEach() is a bit of ambiguous no?